infra/modules/dms-networking/main.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  our_target_cidr_block   = "10.0.0.0/20"   # our [Nava] cidr block, where the target database for the DMS is located
```

Comment:
OH! This, this is wrong. It's supposed to be the default CIDR block still. I got ahead of myself.

Reply:
fixed: https://github.com/HHS/simpler-grants-gov/pull/1043/commits/a9cd45e9086d015748bfe32102827902dbd080e3